app/javascript/esbuild.js

```diff
@@ -0,0 +1,47 @@
+#!/usr/bin/env node
+const fs = require('fs')
+const ImportGlobPlugin = require('esbuild-plugin-import-glob')
+const browserslistToEsbuild = require('browserslist-to-esbuild')
+
+function build() {
+  const env = require('./.config/env.json')
+  require('esbuild')
+    .build({
+      entryPoints: [
+        './app/javascript/packs/application.tsx',
+        './app/javascript/packs/core.tsx',
+        './app/javascript/packs/internal.tsx',
+        ...(process.env.RAILS_ENV === 'test'
+          ? ['./app/javascript/packs/test.tsx']
+          : []),
+      ],
+      bundle: true,
```

Comment:
Is bundling something we want in development?

Reply:
Yeah, I think so. Let's keep that consistent.